### PR TITLE
[script] Update build_all_plugins_app to exclude some plugins in `master`.

### PR DESCRIPTION
**Pull request by ditman**
The latest push to `master` introduced a small dependency version mismatch when building the `all_plugins` app (using camera and google_sign_in together).

This update to the build_all_plugins_app script adds a second exclusion list, to remove conflicting plugins from the builds in `master`, while those get migrated to null safety.

The current exclusion list only adds "camera", and the goal is to keep that exclusion list empty, but others can be added there as we migrate more plugins to null safety.

Complements https://github.com/flutter/plugins/pull/3329

## Pre-launch Checklist

- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: script/nnbd_plugins.sh [hunk 1]
```diff
@@ -21,4 +21,22 @@ readonly NNBD_PLUGINS_LIST=(
...
+  # "image_picker"
+  # "in_app_purchase"
+  # "quick_actions"
+  # "sensors"
+  # "shared_preferences"
+  # "wifi_info_flutter"
+)
+
 export EXCLUDED_PLUGINS_FROM_STABLE=$(IFS=, ; echo "${NNBD_PLUGINS_LIST[*]}")
+export EXCLUDED_PLUGINS_FROM_MASTER=$(IFS=, ; echo "${NON_NNBD_PLUGINS_LIST[*]}")
```
Comment (cyanglaz) [inline script/nnbd_plugins.sh:42]: NL EOF

**Review by cyanglaz — APPROVED** [review]
LGTM assuming test passes.

File: script/nnbd_plugins.sh (hunk 1 above)
Comment (ditman) [reply script/nnbd_plugins.sh]: Yep, done!

**Comment by ditman** [comment]
Merging on red in an attempt to make the tree green again.

**Pull request merged**